app/views/practices/show/search_terms/_category.html.erb

```diff
@@ -1,5 +1,5 @@
 <a
-  class="dm-tag--big--action-primary"
+  class="<%= params[:controller] === 'practices' && params[:action] === 'show' ? 'usa-tag text-uppercase text-semibold' : 'dm-tag--big--action-primary' %>"
```

Comment:
I'm wondering if we need the conditional here. Would this ever be something other than the practices controller with the show action?

Reply:
Good point. The only reason I added it is because I wasn't sure if we'd be using this partial for something else along with the big styling in the future. I went ahead and removed the condition! 😄